tasks/setup_RedHat.yml

```
@@ -17,21 +17,17 @@
       ternary(ansible_distribution + 'Linux', ansible_distribution) }}
 
 - name: Add the Okta ASA repo
-  ansible.builtin.copy:
-    # yamllint complains about the length of a few of the lines below,
+  ansible.builtin.yum_repository:
+    # yamllint complains about the length of a couple of the lines below,
     # but there is no way to shorten them.
     #
     # yamllint disable rule:line-length
```

I think you can remove the `yamllint` disable directive...
```suggestion
  ansible.builtin.yum_repository:
```